From 0bcea4d97bdbfba5920ceb56215a911c9d529dc6 Mon Sep 17 00:00:00 2001
From: Richard Purdie <richard.purdie@linuxfoundation.org>
Date: Wed, 29 May 2019 00:10:32 +0100
Subject: [PATCH] strace: Tweak ptest disk space management

If the test is successful, remove the log and exp files. This stops strace
using around 600MB of disk space and running our ptest images out of space.

RP 2019/5/29

Upstream-Status: Inappropriate [specific to OE image space issues]
---
 tests/gen_tests.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/gen_tests.sh b/tests/gen_tests.sh
index 2a7fada..d71e1c1 100755
--- a/tests/gen_tests.sh
+++ b/tests/gen_tests.sh
@@ -93,6 +93,7 @@ while read -r name arg0 args; do {
 		$hdr
 		. "\${srcdir=.}/init.sh"
 		run_strace_match_diff $arg0 $args
+		rm -rf log exp out
 		EOF
 		;;
 
